### PR TITLE
Created new Matrices overview

### PR DESCRIPTION
**Pull request by ekiernan**

### CODE REVIEW

File: content-v2/guides/userguides/exploring-projects.md [hunk 1]
```diff
@@ -76,20 +76,18 @@ A partial example of a TSV file is listed below:
...
 ## Downloading Project DCP Generated Matrices
 
-Each project processed with DCP pipelines has DCP-generated matrices.
-
-To download Project matrices, navigate to the Project Information page and select the **"Project Matrices"** tab to the left. 
+Each project processed with DCP pipelines has DCP-generated matrices.To download Project matrices, navigate to the Project Information page and select the **"Project Matrices"** tab to the left. 
 
 Scroll to identify the relevant matrix and then select the download icon. 
 
 ![Project Matrices](../_images/dcp_generated_matrices.png "Project Matrices")
```
Comment (kbergin) [inline content-v2/guides/userguides/exploring-projects.md:83]: Is this image going to change to one for DCP2 that has only the loom downloads? ie here https://data.humancellatlas.org/explore/projects/4a95101c-9ffc-4f30-a809-f04518a23803/expression-matrices

File: content-v2/guides/userguides/matrices.md [hunk 2]
```diff
@@ -5,77 +5,80 @@ title: "Matrices"
...
+| `input_name` | Metadata values for `sequencing_input.biomaterial_core.biomaterial_id` |
 
+More information about HCA post-processing for the project-level matrices can be found in the Matrix Overview for the [Optimus pipeline](https://broadinstitute.github.io/warp/documentation/Pipelines/Optimus_Pipeline/Loom_schema.html#hca-data-coordination-platform-matrix-processing) and the [Smart-seq2 Pipeline](https://broadinstitute.github.io/warp/documentation/Pipelines/Smart-seq2_Multi_Sample_Pipeline/Loom_schema.html#table-2-column-attributes-cell-metrics)(in development). 
 
-#### DCP Library-level Matrices (10x Data Only)
-Library-level matrices (Loom files) are only applicable to 10x datasets. They contain the same gene (row) metrics, cell (column) metrics and counts as the project-level matrices, but are separated by the library preparation metadata (the field `sequencing_process.provenance.document_id`).
 
-These matrices do not contain all the species, organ, and library HCA metadata that is included in the project-level matrices. Instead, they only contain the metadata for `input_id` and `input_name`. 
+#### DCP Library-level Matrices
```
Comment (kbergin) [inline content-v2/guides/userguides/matrices.md:63]: Could we add a line about why someone would want to use the library level matrices? I'd say the two main reasons would be if they are looking for unfiltered results, and if they are looking to only pull matrices for certain subsets of a project/projects and don't want all of the data.

**Review by kbergin — CHANGES_REQUESTED** [review]

File: content-v2/guides/userguides/exploring-projects.md (hunk 1 above)
Comment (ekiernan) [reply content-v2/guides/userguides/exploring-projects.md:83]: @kbergin We did update it- it looks like it renders appropriately on the dev environment https://dev.singlecell.gi.ucsc.edu/guides

File: content-v2/guides/userguides/matrices.md (hunk 2 above)
Comment (ekiernan) [reply content-v2/guides/userguides/matrices.md:63]: Yes, that would be very helpful. Will add.

File: content-v2/guides/userguides/exploring-projects.md (hunk 1 above)
Comment (kbergin) [reply content-v2/guides/userguides/exploring-projects.md:83]: Oh great, ok! It must have just rendered wrong on github. Thanks!

**Comment by NoopDog** [comment]
closing will merge manually to master when we deploy to prod.

**Pull request closed without merging**